Expensify/App
# Update validate page in VBA flow

### Details
Add a confirmation button to the VBA Validation page and remove the animation image.

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/176433

### Tests


- Create a new user
- Create workspace via the global create menu
- Select 'Get Started' after the workspace is created
- Go through VBA setup using the test credentials in (2), and arrive at Validate step
- **There should be no animated image**
- **There should be a button at the bottom marked 'Got it'**
- **Tapping the button should close the current page**


### QA Steps

Run above test

### Tested On


### Screenshots


#### Web

![Screenshot 2021-09-14 at 14 47 27](https://user-images.githubusercontent.com/10736861/133425448-9e99e4fb-5838-467e-ae83-2e0a1823aeff.png)


#### Mobile Web

![Simulator Screen Shot - iPhone 8 - 2021-09-15 at 12 18 18](https://user-images.githubusercontent.com/10736861/133425365-55c37476-9de3-4dc2-bdad-8260979af5cc.png)


#### Desktop

<img width="1202" alt="Screenshot 2021-09-15 at 11 34 09" src="https://user-images.githubusercontent.com/10736861/133425404-34b80056-7eac-4293-843e-00146e03e92d.png">


#### iOS

![Simulator Screen Shot - iPhone 8 - 2021-09-15 at 12 15 27](https://user-images.githubusercontent.com/10736861/133425356-ff8579ec-7b4c-4209-8fc6-6f2d5084c3c8.png)


#### Android

![Screenshot_1631705201](https://user-images.githubusercontent.com/10736861/133425471-875c554e-517b-4edf-9fee-cea0e857d92d.png)
